<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{

background-color:transparent;
color: #aa54ff; 
}
#filter,#mediaplayer,#notif,#pdf,#supprimer,#search,#statistics{
background-color: transparent;
}


#widget{
background-color: #040f13;
}
#sidemenu{
background-color: #071e26; 
border-radius: 20px; 
}
#ajouter,#modifier
{
padding: 20px; 
background-color: #7f3fbf; 
border-radius: 5px;
color : #fff;
}
QPushButton,QLineEdit{
padding: 20px; 
background-color: #040f13; 
border-radius: 5px; }
#header{
background-color: #071e26; 
border-radius: 10px;
}
#rightsidemenu,#popupLabel
{
background-color: #071e26; 
border-radius: 20px; 
}

#mainbody{
background-color: #071e26; 
border-radius: 10px;
}
#comboBox{

}

</string>
  </property>
  <widget class="QFrame" name="header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>50</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../Downloads/hamburger.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>9</y>
      <width>72</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>20</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4"/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>10</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>50</horstretch>
      <verstretch>10</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/hamburger.png</normaloff>icons/hamburger.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>20</height>
     </size>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>50</horstretch>
      <verstretch>50</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>1</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>InteLab</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/notification.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/video.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="mediaplayer">
    <property name="geometry">
     <rect>
      <x>954</x>
      <y>12</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>mediaPlayer</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/play-button.png</normaloff>icons/play-button.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="notif">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="popupLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_12</zorder>
   <zorder>notif</zorder>
   <zorder>mediaplayer</zorder>
   <zorder>frame</zorder>
   <zorder>frame_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>label</zorder>
   <zorder>label_11</zorder>
   <zorder>popupLabel</zorder>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>1191</width>
     <height>631</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="sidemenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>221</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>211</width>
       <height>531</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>  encadrant</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/teacher.png</normaloff>icons/teacher.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>etudiant</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/student.png</normaloff>icons/student.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>materiel</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/flask.png</normaloff>icons/flask.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>sponsor</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/handshake.png</normaloff>icons/handshake.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>  publication</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/research.png</normaloff>icons/research.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pushButton_5</zorder>
     <zorder>pushButton_6</zorder>
     <zorder>pushButton_4</zorder>
    </widget>
   </widget>
   <widget class="QFrame" name="mainbody">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>10</y>
      <width>711</width>
      <height>601</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Liste des materiels</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rechercher">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>20</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/search (1).png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/funnel.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="filter">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>510</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/pdf.png</normaloff>icons/pdf.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>220</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/pdf (1).png</pixmap>
     </property>
    </widget>
    <widget class="QTableView" name="table_materiel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>571</width>
       <height>471</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_supp">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>50</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/garbage-bin.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/search (1).png</normaloff>icons/search (1).png</iconset>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>id</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>quantite</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>140</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/pie-chart.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="statistics">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>460</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>STA</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/statistic.png</normaloff>icons/statistic.png</iconset>
     </property>
    </widget>
    <zorder>label_15</zorder>
    <zorder>rechercher</zorder>
    <zorder>label_2</zorder>
    <zorder>label_13</zorder>
    <zorder>label_14</zorder>
    <zorder>table_materiel</zorder>
    <zorder>id_supp</zorder>
    <zorder>label_16</zorder>
    <zorder>supprimer</zorder>
    <zorder>search</zorder>
    <zorder>filter</zorder>
    <zorder>comboBox</zorder>
    <zorder>label_17</zorder>
    <zorder>statistics</zorder>
    <zorder>pdf</zorder>
   </widget>
   <widget class="QFrame" name="rightsidemenu">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>211</width>
      <height>611</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>201</width>
       <height>371</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="le_id">
        <property name="placeholderText">
         <string>  Identifiant</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_nom">
        <property name="placeholderText">
         <string>      Nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="la_qte">
        <property name="placeholderText">
         <string>   Quantité</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="la_disp">
        <property name="placeholderText">
         <string> Disponibilite</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_prix">
        <property name="placeholderText">
         <string>      Prix</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>440</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter un materiel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>490</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
   </widget>
   <zorder>mainbody</zorder>
   <zorder>sidemenu</zorder>
   <zorder>rightsidemenu</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
